interpreter/cling/include/cling/Interpreter/Interpreter.h

```diff
@@ -12,8 +12,6 @@
 
 #include "cling/Interpreter/InvocationOptions.h"
 
-#include "llvm/ADT/StringRef.h"
```

Comment:
It's likely missing many removals. Given that `StringRef.h` was the only llvm header we `#include`d in `Interpreter.h` I'd expect that we can remove *all* llvm headers from that list! I.e. empty it. I.e. remove that whole feature.